### PR TITLE
remove goma support from clang

### PR DESCRIPTION
**Pull request by atetubou**
goma is deprecated and not maintained anymore.

https://chromium.googlesource.com/infra/goma/client/

### CODE REVIEW

**Review by AaronBallman — APPROVED** [review]
LGTM!

**Comment by atetubou** [comment]
I don't have merge permission. Could someone merge this PR if this looks good?

**Pull request merged**
